LuxHeader    header

#Ibeo Fusion vehicle state 2806 & 2807
uint64        time_stamp
int32         distance_x
int32         distance_y
float32       course_angle              # rad
float32       longitudinal_velocity
float32       yaw_rate
float32       steering_wheel_angle
float32       front_wheel_angle         # rad
float32       vehicle_width             # m
float32       vehicle_front_to_front_axle
float32       rear_axle_to_front_axle
float32       rear_axle_to_vehicle_rear
float32       steer_ratio_poly0
float32       steer_ratio_poly1
float32       steer_ratio_poly2
float32       steer_ratio_poly3
float32       longitudinal_acceleration          # only in 2807
